payments-core/src/main/java/com/stripe/android/CustomerSession.kt

```diff
@@ -27,6 +27,10 @@ import kotlin.coroutines.CoroutineContext
  *
  * See [Creating ephemeral keys](https://stripe.com/docs/mobile/android/standard#creating-ephemeral-keys)
  */
+@Deprecated(
+    "Please use Mobile Payment Element instead. If you already using CustomerSession, learn how to migrate" +
```

Comment:
Can we add this string as a constant, and reference it in all of these?

Reply:
yep! Done, thanks